### PR TITLE
Minor corner calibration changes: logging and version added in pic_se…

### PR DESCRIPTION
**Pull request by andywerner**
…ttings.yaml

### CODE REVIEW

**Comment by andywerner** [comment]
@amrsoll 
Is there somewhere some documentation what data is for what in the pic_settings.yaml?
Looks like there are many deprecated data in there...
If not, please add some to the code.


```
blur_factor_threshold: 20
calibMarkers:
  NE: [1967, 256]
  NW: [120, 217]
  SE: [1902, 1392]
  SW: [137, 1361]
calibration_updated: false
cornersFromImage:
  NE: [1914, 124]
  NW: [171, 90]
  SE: [1833, 1453]
  SW: [177, 1420]
hostname_KEY: MrBeam-ANDY
marker2cornerVecs:
  NE: [-53, -132]
  NW: [51, -127]
  SE: [-69, 61]
  SW: [40, 59]
markerSettings:
  hueMax: {NE: 180, NW: 180, SE: 180, SW: 180}
  hueMin: {NE: 130, NW: 130, SE: 130, SW: 130}
  hue_lb_min: 100
  pixels_expected: 800
  radMax: 23
  radMin: 3
  ratioH: 4
  ratioW: 4
  saturationMax: {NE: 255, NW: 255, SE: 255, SW: 255}
  saturationMin: {NE: 0, NW: 0, SE: 0, SW: 0}
  valueMax: {NE: 255, NW: 255, SE: 255, SW: 255}
  valueMin: {NE: 0, NW: 0, SE: 0, SW: 0}
raw_calibMarkers:
  NE: [1871, 203]
  NW: [180, 296]
  SE: [1883, 1259]
  SW: [271, 1328]
raw_cornersFromImage:
  NE: [1811, 111]
  NW: [224, 206]
  SE: [1813, 1302]
  SW: [341, 1349]
version: 0.1.15
```

File: octoprint_mrbeam/camera/corners.py [hunk 1]
```diff
@@ -20,9 +20,9 @@
 from numpy.linalg import norm
 import cv2
 from octoprint_mrbeam.camera import lens
+from octoprint_mrbeam.mrb_logger import mrb_logger
 
-# Set this after merging the logging overhaul.
-_logger = logging.getLogger(__name__)
+_logger = mrb_logger("octoprint.plugins.mrbeam.camera.corners")
```
Comment (amrsoll) [inline octoprint_mrbeam/camera/corners.py:25]: Please keep the consistent `__name__` scheme

**Review by amrsoll — CHANGES_REQUESTED** [review]
We should do something about the logging because it creates a lot of pollution in the tests as well. I still believe we should have a 2nd look at the logging overhaul I made previously #947 

Also the deprecated values that you do find in the yaml file is just carried over from the previous version, they are not actively saved.

File: octoprint_mrbeam/camera/corners.py (hunk 1 above)
Comment (andywerner) [reply octoprint_mrbeam/camera/corners.py:25]: this leads to inconsistent log output. 
`__name__` results in `octoprint_mrbeam.camera.corners` while all the other mr beam output is this:
`octoprint.plugins.mrbeam.iobeam.lidhandler`

Comment (andywerner) [reply octoprint_mrbeam/camera/corners.py:25]: I agree that `__name__` is nicer and that even it's output is shorter (= nicer). @amrsoll would you like to go though the code and change all logger names at once? (after the next stable?)

Comment (andywerner) [reply octoprint_mrbeam/camera/corners.py:25]: > Also the deprecated values that you do find in the yaml file is just carried over from the previous version, they are not actively saved.

Yes, that's why we need some basic documentation.

Comment (amrsoll) [reply octoprint_mrbeam/camera/corners.py:25]: Yeah sure I can change the naming later :+1:

Comment (amrsoll) [reply octoprint_mrbeam/camera/corners.py:25]: ok - we don't have documentation on those yet.

**Review by amrsoll — APPROVED** [review]

**Pull request merged**